settings.yml

```diff
@@ -165,10 +165,6 @@ grid_items_size: large
 #       - <CATEGORY1_NAME>
 #       - <CATEGORY2_NAME>
 #
-groups:
```

Comment:
Why removing this setup?. It seems that We need to add here the groups and categories you used in data.yml

Please check the documentation of the landscape project. Check the reference docs here: https://github.com/cncf/landscape2/blob/main/docs/config/settings.yml

Reply:
I understand the need to add groups to the setup. However, I believe the current group and category structure in might be incorrect.

With the current implementation, the default group seems to be named by its only category, "Adopters". Additionally, it only shows the "adopters" category. If we add another group, like "sponsors" with its own category, it wouldn't be displayed on the default page.

Therefore, I propose the following:

- Create a default group with a more appropriate name.
- Include both "adopters" and "sponsors" categories within this default view.

Reply:
How about we make the default view group to be "JSON Schema Ecosystem" and it would have all categories, and we can add more specific groups later with appropriate categories as we grow.

Reply:
Thanks DV. Try that and lets see the result.